### ISSUE TITLE
Elasticsearch 5 has maybe more reason to allow the creation of mapping

### ISSUE THREAD

**Issue by kootenpv** [knockrentals/scrapy-elasticsearch#61]
Maybe it would be possible to ingest a mapping into elasticsearch before writing items? E.g. 

    ELASTICSEARCH_MAPPING = { ... some json ...}

or

    ELASTICSEARCH_MAPPING = "/file/to/mapping.json"

It'd be a nice flow to have it be included at this stage.

**Comment by jayzeng** [comment]
@kootenpv yeah, I will consider to implement this. https://github.com/knockrentals/scrapy-elasticsearch/issues/51 also asks for the same feature.
